subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -391,6 +394,7 @@ struct node_rx_iso_meta {
 	uint64_t payload_number:39; /* cisPayloadNumber */
 	uint64_t status:8;          /* Status of reception (OK/not OK) */
 	uint32_t timestamp;         /* Time of reception */
+	void     *next;             /* Pointer to next pre-transmission rx_node (BIS) */
```

Comment:
Can you explain the purpose, how/when allocated/assigned and released/reset?

Reply:
Yes, we have opted to not use the `lll->payload[]` array in our LLL implementation. In our LLL, payloads are delivered to ISO-AL from either payload subevent reception or from a linked pre-transmission list. In a later PR I will try to make some memory saving from conditionally compiling in `payload[]`.

All received pre-transmissions are linked from "head" to last via this new next-pointer. This saves the overhead of a FIFO or memq. You can easily argue that this could be part of some "vendor" struct, but residing inside `node_rx_iso_meta` makes it "free" in terms of RAM usage, due to the larger `node_rx_ftr` in the union.

To avoid the "dangling" variable seemingly not used in the controller, the comment could be changed to something like: `/* For vendor LLL node lists */`, similar to `next` in `node_rx_hdr`.

Any other suggestions or concerns?

Reply:
Thanks for the explanation. No concerns, just curious. And yes, feel free to abstract or optimise unused vendor struct members as necessary.